packages/block-library/src/paragraph/block.json

```diff
@@ -15,7 +15,12 @@
 			"type": "rich-text",
 			"source": "rich-text",
 			"selector": "p",
-			"role": "content"
+			"role": "content",
+			"control": {
+				"type": "RichText",
```

Comment:
Rather than specifying the type of control that the attribute needs, I wonder if we should be more semantic, and let Gutenberg decide what control to display for each different piece of semantic content, for example:

- Media
- Link
- Text
- etc.